File: js/Sort/bubble.sort.js
const arr = [102,3,402,350,20];


console.log("Original array is: ",arr)

for(let i = 0; i<arr.length; i++){
    for(let j = i; j<arr.length; j++){
        if(arr[i] > arr[j]){
            [arr[i],arr[j]] = [arr[j],arr[i]]
        }
    }
}


console.log("Sorted array is: ",arr)

File: python/sort/bubble.sort.py
arr = [40,20,403,504,694,209]

size = len(arr)

print("Original Array is: ",arr)

for i in range(size-1):
    for j in range(size-1-i):
        if arr[j] > arr[j + 1]:
            arr[j],arr[j + 1] = arr[j + 1],arr[j]
            
            
print("Sorted Array is: ",arr)